app/controllers/admin/loan_questions_controller.rb

```diff
@@ -0,0 +1,78 @@
+class Admin::LoanQuestionsController < Admin::AdminController
+  include TranslationSaveable
+  before_action :set_loan_question, only: [:show, :edit, :update, :destroy, :move]
+
+  def index
+    authorize CustomField
+    @questions = CustomField
+        .joins(:custom_field_set)
+        .where(custom_field_sets: {internal_name: ['loan_criteria', 'loan_post_analysis']})
+    @json = ActiveModel::Serializer::CollectionSerializer.new(@questions.roots).to_json
+  end
+
+  def new
+    field_set_name = params[:fieldset]
+    field_set = CustomFieldSet.find_by(internal_name: 'loan_' + field_set_name)
+    @loan_question = field_set.custom_fields.build
+    authorize @loan_question
+    render partial: 'form'
+  end
+
+  def edit
+    render partial: 'form'
+  end
+
+  def create
+    @loan_question = CustomField.new(loan_question_params)
+    authorize @loan_question
+    if @loan_question.save
+      render json: @loan_question.reload
+    else
+      render partial: 'form', status: :unprocessable_entity
+    end
+  end
+
+  def update
+    if @loan_question.update(loan_question_params)
+      render json: @loan_question.reload
+    else
+      render partial: 'form', status: :unprocessable_entity
+    end
+  end
+
+  def move
+    target = CustomField.find(params[:target])
+    method = case params[:relation]
+      when 'before' then :prepend_sibling
+      when 'after' then :append_sibling
+      when 'inside' then :prepend_child
+    end
+
+    target.send(method, @loan_question)
+    head :no_content
+  rescue
```

Comment:
The ajax request would still receive a 500 response and the page should then say 'server error'. If it's not a known error, that's all we should be reporting to the user. They can then tell us it errored, and we can look into it. When was the last time you saw an error page on Google that included technical stuff like `nil` this or that? They just say 'server error'. I feel pretty strongly about this one.